tests/gpts/github/test-config/github_positive_suite.ts

```diff
@@ -195,7 +195,20 @@ export const gitHubBasicGoldenPathTemplateTests = (gptTemplate: string) => {
                 expect(finished).toBe(true)
             }
         }, 900000)
-
+  
+         /**
+         * Check if the pipelinerun yaml has the rh-syft image path mentioned
+         */
+         it(`Check ${gptTemplate} pipelinerun yaml has the rh-syft image path`, async ()=> {
+            const pipelineRun = await kubeClient.getPipelineRunByRepository(repositoryName, 'push')
+            if (pipelineRun && pipelineRun.metadata && pipelineRun.metadata.name) {
+                const doc = await kubeClient.pipelinerunfromName(pipelineRun.metadata.name,developmentNamespace)
+                const image_name = doc.spec.pipelineSpec.tasks[2].taskSpec.steps[1].image
+            console.log(image_name)
```

Comment:
I would delete this or at least improve the output: "Image used in pipelinereun:  "+...

Reply:
Updated the output for readability